api/groups/baseAccountsGroup_test.go

```diff
@@ -423,10 +443,63 @@ func TestGetESDTTokens_ReturnsSuccessfully(t *testing.T) {
 	loadResponse(resp.Body, &shardResponse)
 
 	assert.Equal(t, http.StatusOK, resp.Code)
-	assert.Equal(t, shardResponse.Data.Tokens, expectedTokens)
+	assert.Equal(t, expectedTokens, shardResponse.Data.Tokens)
 	assert.Empty(t, shardResponse.Error)
 }
 
+func TestGetGuardianData(t *testing.T) {
```

Comment:
Just to keep a line with other test cases:
add `/ ---- GetGuardianData`

Reply:
done